infrastructure/terraform/README.md

```diff
@@ -5,7 +5,7 @@
 - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) installed
 - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed
 
-## Setup
+## Git Setup
 
 ```
 git config --global submodule.recurse true
```

Comment:
These line need to be swapped.

git submodule update --init --recursive

git clone  https://github.com/Azure/ai-hub.git